erpnext/manufacturing/doctype/routing/routing.py

```diff
@@ -11,6 +11,15 @@
 class Routing(Document):
 	def validate(self):
 		self.set_routing_id()
+		self.update_op_time()
+
+	def update_op_time(self):
+		bom_list = frappe.db.sql("""select name, routing from `tabBOM`
```

Comment:
No need to select `routing` field. Also you can replace this with ORM function, something like:

```python
frappe.get_list("BOM", filters={"routing": self.name}, pluck="name")
```